Added documentation (#27)
iotaledger/inx-participation: documentation/docs/welcome.md

---
description: INX-Participation is an extension for nodes to enable on-tangle voting.
image: /img/Banner/banner_hornet.png
keywords:
- IOTA Node
- Hornet Node
- INX
- Participation
- IOTA
- Shimmer
- Node Software
- Welcome
- explanation
---

# Welcome to INX-Participation

> INX-Participation is an extension for nodes to enable on-tangle voting. 

Reviewer: Could you please be more specific? How exactly does it enable the voting? Who is voting?

For example,

"INX-Participation extension handles voting requests from the connected clients and calculates the current voting status for them."

> The extensions maintains its own database separate from that of the node and provides means to track events and votes.

Reviewer: ```suggestion
The extensions maintains its own database separate from that of the node and provides means to track events and votes.
```

The database part sounds like a detail of implementation. I would move it to the very end of the paragraph or remove it altogether if it is not actually important to know.

> You can find more information on how participation events work in the [HORNET Participation Plugin](https://github.com/iota-community/treasury/blob/main/specifications/hornet-participation-plugin.md) specification.

Reviewer: ```suggestion
You can find more information about participation events in the [HORNET Participation Plugin](https://github.com/iota-community/treasury/blob/main/specifications/hornet-participation-plugin.md) specifications.
```

> You can find more information on how participation events work in the [HORNET Participation Plugin](https://github.com/iota-community/treasury/blob/main/specifications/hornet-participation-plugin.md) specification.

Reviewer: Is it HORNET or Hornet? I am seeing both versions across the wiki.

@lucas-tortora

Reviewer 2: Unless it's an acronym, which I don't think is the case, I think it should be Hornet.

## Setup

The recommended setup is to use the provided [Docker images](https://hub.docker.com/r/iotaledger/inx-participation).
These images are also used in our [HORNET recommended setup using Docker](http://wiki.iota.org/hornet/develop/how_tos/using_docker).

## Configuration

> The participation extension is configured by default to connect to your HORNET instance.

Reviewer: ```suggestion
The participation extension connects to the running HORNET instance by default.
```

Active voice in general sounds more straightforward than passive voice.

You can find all the configuration options in the [configuration section](configuration.md).

## Dashboard

If you are using the [INX-Dashboard](https://github.com/iotaledger/inx-dashboard) on your node, you can manage events directly from your browser.

## API

> The extension exposes a custom set of REST APIs that can be used by wallets and applications to find active, past and upcoming participation events and query event results.

Reviewer: ```suggestion
The extension exposes a custom set of REST APIs that can be used by wallets and applications to find past, active, and upcoming participation events and query event results.
```
